tools/cloud-build/daily-tests/tests/omnia.yml

```diff
@@ -19,6 +19,6 @@ zone: us-central1-c
 workspace: /workspace
 blueprint_yaml: "{{ workspace }}/community/examples/omnia-cluster.yaml"
 blueprint_dir: blueprint-omnia
-network: "default"
```

Comment:
It used to use the default network.  I am wondering if the change was intentional...